rest-api-spec/build.gradle

```diff
@@ -74,4 +74,6 @@ tasks.named("yamlRestCompatTestTransform").configure ({ task ->
   task.skipTest("index/91_metrics_no_subobjects/Metrics object indexing with synthetic source", "_source.mode mapping attribute is no-op since 9.0.0")
   task.skipTest("index/91_metrics_no_subobjects/Root without subobjects with synthetic source", "_source.mode mapping attribute is no-op since 9.0.0")
   task.skipTest("indices.create/20_synthetic_source/synthetic_source with copy_to inside nested object", "temporary until backported")
+  task.skipTest("update/10_synthetic_source/keyword", "synthetic source recovery behind a feature flag and not enabled in release builds")
```

Comment:
I had to skip these yaml test here because they expect synthetic source recovery to be used.